testing/internal/e2e/boundary/account.go

```diff
@@ -0,0 +1,75 @@
+package boundary
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/accounts"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/go-secure-stdlib/base62"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateNewAccountApi creates a new account using the go api.
+// Returns the id of the new account as well as the password that was generated
+func CreateNewAccountApi(t testing.TB, ctx context.Context, client *api.Client, loginName string) (string, string) {
```

Comment:
In situations like this it can help to name the return parameters
```suggestion
func CreateNewAccountApi(t testing.TB, ctx context.Context, client *api.Client, loginName string) (accountId string, password string) {
```